components/ai_chat/core/browser/conversation_driver.cc

```diff
@@ -1101,14 +1137,20 @@ bool ConversationDriver::HasPendingConversationEntry() {
 }
 
 int ConversationDriver::GetContentUsedPercentage() {
-  if (GetCurrentModel().max_page_content_length >
+  auto& model = GetCurrentModel();
+  if (!model.option->is_leo_model()) {
+    return 100;
```

Comment:
If we're using `kCustomModelMaxPageContentLength` shouldn't we use that here?

Reply:
ah, missed this place. added `kCustomModelMaxPageContentLength`